mkdocs.yml

```
@@ -88,6 +88,8 @@ nav:
           - test_framework_integration/junit_5.md
           - test_framework_integration/spock.md
           - test_framework_integration/manual_lifecycle_control.md
+    - Samples:
+        - samples.md
```

```suggestion
    - Examples: examples.md
```
So we don't create a sub-page (we currently only have one level).